src/execution.jl

```diff
@@ -62,6 +62,54 @@ end
 
 ## cached compilation
 
+### Disk cache notes
```

Comment:
Name bikeshedding: including `disk` is kinda specific, but "Persistent cache" is a bit much.